docs/plugins/form-builder.mdx

```diff
@@ -430,10 +430,26 @@ formBuilderPlugin({
         plural: 'Custom Text Fields',
       },
     },
+    country: {
```

Comment:
Add a comment explaining // example overridding the generated type for this block
And change CountrySelect to CountryFormBlock
__Select can be generated from a collection too for our select API